roles/storage_nfs_lvm/README.md

```diff
@@ -0,0 +1,104 @@
+# storage_nfs_lvm
```

Comment:
Since this is openshift specific, the role should have openshift in the name.

Consider renaming this role `openshift_storage_nfs_lvm`